docs/installation/prerequisites-installation.md

```diff
@@ -1,18 +1,19 @@
-# Installation on local PC
-
 [Back](../README.md)
 
-## Prerequisites for Development Workplace
+## Prerequisites Installation
+
+In the description below, the following prerequisites are installed:
 
 * github account to login to projectbuilder
 * Java 14 or later
 * Maven 3
-* NodeJs with NPM - [NodeJS Install](https://nodejs.org/en/download/package-manager/)
+* NodeJs with NPM (version 8.3.2) - [NodeJS Install](https://nodejs.org/en/download/package-manager/)
+* docker (Version 20.10.8 or higher) and docker compose (version 1.29.2)
```

Comment:
Is there a reason why 20.10.8 is given?

Reply:
Ne, aber die Version sieht mir als min requirement aus der Luft gegriffen aus. Es könnte ja auch eine vorher gehen

Reply:
Würde dann wenn schreiben, dass es mit der Version getestet wurde, aber nicht ein min requirement ist